site/en/docs/extensions/reference/commands/index.md

```diff
@@ -279,7 +273,7 @@ of commands returned by `commands.getAll()`.
 {% endAside %}
 
 ```js
-// background.js
+// service-worker.js
```

Comment:
Can we convert this comment to a [label](https://developer.chrome.com/docs/handbook/components/#labels)?